### ISSUE TITLE
linux compatibility 

### ISSUE THREAD

**Issue by CasualHubEnjoyer** [tremwil/SteamP2PInfo#45]
is there a way to make this work on linux?

**Comment by tremwil** [comment]
No, not that I know of sorry. I suspect this is impossible as the ping monitoring relies on Windows Kernel Event Tracing to capture packets that the Steam process sends. I suspect the Linux steam client sends packets directly from the Linux steam process, thereby preventing packets from being intercepted whether or not Wine supports Windows Event Tracing.

**Comment by tremwil** [comment]
However, it should be possible to make a native Linux app with equivalent functionality. You'd have to switch to .NET core and overhaul the code used to intercept udp packets, but it would be possible.

**Comment by CasualHubEnjoyer** [comment]
Thanks for the quick reply! That's tragic, your apps been a staple but linux seems to have better performance. I'm in way over my head as is but I appreciate the response!
